content/source/docs/cloud/api/oauth-clients.html.md

```diff
@@ -200,10 +200,13 @@ Key path                             | Type   | Default | Description
 `data.type`                          | string |         | Must be `"oauth-clients"`.
 `data.attributes.service-provider`   | string |         | The VCS provider being connected with. Valid options are `"github"`, `"github_enterprise"`, `"gitlab_hosted"`, `"gitlab_community_edition"`, `"gitlab_enterprise_edition"`, or `"ado_server"`.
 `data.attributes.name`               | string | `null`  | An optional display name for the OAuth Client. If left `null`, the UI will default to the display name of the VCS provider.
+`data.attributes.key`                | string |         | The OAuth Client key. It sometimes refers to a Consumer Key or Application Key or some other client key for the VCS provider.
```

Comment:
Also - should customer key and application key be capitalized? If they're like official terms then yes, but I wasn't sure if they were as we don't refer to a specific provider or anything. 
```suggestion
`data.attributes.key`                | string |         | The OAuth Client key. It can refer to a Consumer Key, Application Key, or another type of client key for the VCS provider.
```